apps/tangle-dapp/components/BridgeTxQueueDropdown/BridgeTxQueueItem.tsx

```diff
@@ -140,22 +151,33 @@ const getActiveStep = (state: BridgeTxState, type: BridgeType): number => {
     type === BridgeType.SYGMA_EVM_TO_SUBSTRATE ||
     type === BridgeType.SYGMA_SUBSTRATE_TO_SUBSTRATE;
 
+  const isHyperlane = type === BridgeType.HYPERLANE_EVM_TO_EVM;
+
   switch (state) {
     case BridgeTxState.Initializing:
       return 1;
     case BridgeTxState.Sending:
       return 2;
-    case BridgeTxState.SygmaIndexing: {
+    case BridgeTxState.SygmaIndexing:
       if (!isSygma) throw new Error('Invalid state for non-Sygma tx');
       return 3;
-    }
-    case BridgeTxState.SygmaPending: {
+    case BridgeTxState.SygmaPending:
       if (!isSygma) throw new Error('Invalid state for non-Sygma tx');
       return 4;
-    }
     case BridgeTxState.Executed:
     case BridgeTxState.Failed:
       return isSygma ? 5 : 3;
+    case BridgeTxState.HyperlanePending:
+    case BridgeTxState.HyperlaneIndexing:
+      if (!isHyperlane) throw new Error('Invalid state for non-Hyperlane tx');
+      return 2;
+    case BridgeTxState.HyperlaneDelivered:
+    case BridgeTxState.HyperlaneExecuted:
+    case BridgeTxState.HyperlaneFailed:
+      if (!isHyperlane) throw new Error('Invalid state for non-Hyperlane tx');
+      return 3;
+    default:
+      throw new Error(`Unhandled BridgeTxState: ${state}`);
```

Comment:
This function is used directly in the UI, so throwing errors here can crash the dApp.